tests/unit/moduleapi/basics.tcl

```diff
@@ -30,5 +30,11 @@ start_server {tags {"modules"}} {
         assert_equal $reply 3
     }
 
+    test {Unsafe command names are sanitized in INFO output} {
+        r test.weird:cmd
+        set info [r info commandstats]
+        assert_match {*cmdstat_test.weird_cmd:calls=1*} $info
+    }
```

Comment:
the "basics" unit is a mess (taken from some test module before there was a framework, many of it's tests are internal assertions rather than TCL).

i'd rather move that test elsewhere, maybe to `misc.tcl`?